### PR TITLE
add sinhala language

### PR DESCRIPTION
**Pull request by MasterBrian99**
add sinhala language json file
modified LanguageSelector.tsx component

### CODE REVIEW

File: kofta/src/app/components/LanguageSelector.tsx [hunk 1]
```diff
@@ -45,6 +45,7 @@ export const LanguageSelector: React.FC<LanguageSelectorProps> = ({
     { value: "pt-PT", label: "🇵🇹 Português (Europeu)" }, // Portuguese (Portugal)
     { value: "ro", label: "🇷🇴 Română" }, // Romanian
     { value: "ru", label: "🇷🇺 Русский" }, // Russian
+    { value: "si", label: "🇱🇰 සිංහල" },
```
Comment (HarrisonMayotte) [inline kofta/src/app/components/LanguageSelector.tsx]: I'd add a // comment there with the language.

**Review by HarrisonMayotte — CHANGES_REQUESTED** [review]
Other than the one comment below, it's all good.

File: kofta/src/app/components/LanguageSelector.tsx (hunk 1 above)
Comment (MasterBrian99) [reply kofta/src/app/components/LanguageSelector.tsx]: fixed it

**Review by HarrisonMayotte — APPROVED** [review]
Looks good! 👍🏼

**Pull request merged**